VideoWeb/VideoWeb.AcceptanceTests/Pages/HearingRoomPage.cs

```diff
@@ -15,6 +15,6 @@ public static class HearingRoomPage
         public static readonly By ToggleHandRaised = By.Id("toggle-hand-raised-img");
         public static readonly By HandRaised = By.ClassName($"fa-hand-paper");
         public static readonly By ParticipantPanel = By.Id($"panelList");
-        public static By InterPreterName(string displayName) => By.XPath($"//span[contains(text(),'{displayName}')]");
+        public static By InterPreterName(string displayName) => By.XPath($"//span[contains(text()[2], '{displayName}')]");
```

Comment:
Would it be worth adding ids to these elements on the UI to make these tests easier?

Reply:
It would be, I asked Simon and kevin if this could be done for all web elements, I'd also want there to be a rule that you don't put two values in one web element as this does: <span>Auto_individual_44<br>Auto_interpreter_04<span> as that makes finding the elements harder and the testing becomes brittle as it could change at any time,

Reply:
if using xpath without using hardcoded index is ok.